src/lib.rs

```diff
@@ -22,24 +22,22 @@ mod types {
     impl TextOrNullableText for Nullable<Text> {}
 
     #[derive(SqlType)]
-    #[postgres(type_name = "regconfig")]
-    pub struct Regconfig;
+    #[diesel(postgres_type(name = "regconfig"))]
+    pub struct RegConfig;
 }
 
 pub mod configuration {
-    use crate::Regconfig;
-
-    use std::io::Write;
+    use crate::RegConfig;
 
     use diesel::backend::Backend;
+    use diesel::backend::RawValue;
     use diesel::deserialize::{self, FromSql};
-    use diesel::serialize::{self, Output};
+    use diesel::serialize::{self, Output, ToSql};
     use diesel::sql_types::Integer;
-    use diesel::types::ToSql;
 
     #[derive(Debug, PartialEq, AsExpression)]
-    #[sql_type = "Regconfig"]
-    pub struct TsConfiguration(pub u32);
+    #[diesel(sql_type = RegConfig)]
+    pub struct TsConfiguration(pub i32);
```

Comment:
Can you add a comment here why the type of the inner field was changed to `i32`?